### DISCUSSION TITLE
How to run jobs on remote cluster?

### DISCUSSION THREAD

**Discussion by johnwkeller54** [cyllab/CalcUS#37] [General]
Where do I enter my password for the remote cluster? How do I specify how many nodes to use in a gaussian calculation, or do I use a SLURM script to start my gaussian calculation on the cluster?

**Comment by RaphaelRobidas** [comment]
You can follow the procedure here: https://calcus.readthedocs.io/en/latest/usage.html#remote-calculations

You won't need directly your cluster password. Instead, CalcUS will generate a public/private SSH key pair and you can manually add the public key in the authorized keys on the remote cluster (normally in `~/.ssh/authorized_keys`).

The submission options are indeed configured in a SLURM script (see in the documentation for the details). CalcUS however needs to know the number of cores and amount of memory per calculation in order to generate the Gaussian input files.
